### DISCUSSION TITLE
How to stop TimePicker from setting the current time as the default value?

### DISCUSSION THREAD

**Discussion by Stephan-Wells-EPAM** [epam/UUI#984] [Q&A]
I am using a [TimePicker](https://uui.epam.com/documents?category=components&id=timePicker&mode=doc&skin=UUI4_promo) component to allow the user to set the time for a form. The field's default value appears to automatically be set to the current time:

![image](https://user-images.githubusercontent.com/104199342/213508837-df058eb0-495c-47b7-ab9a-167d3b049b09.png)

I want it to be empty by default:

![image](https://user-images.githubusercontent.com/104199342/213509040-70e71d23-af9f-4e82-acff-04ccbc6a4490.png)

How can this be achieved? Thank you.

**Comment by AlekseyManetov** [comment]
Hi, try to pass `null` as a value to TimePicker:
https://codesandbox.io/s/uui-forked-xip8jx?file=/Example.tsx

**Reply by Stephan-Wells-EPAM** [reply]
@AlekseyManetov Thank you for the response! Unfortunately when I try that I get a type error from Typescript:

![image](https://user-images.githubusercontent.com/104199342/214240656-52b45922-4039-4004-8e98-aea318e655e5.png)

**Reply by AlekseyManetov** [reply]
We will fix this typings in next release, as a workaround you can cast value type to any - `value={value as any}` and remove it after feature update

**Reply by Stephan-Wells-EPAM** [reply]
Great to hear. Thank you!
